docs/3-transfer.md

```diff
@@ -270,6 +270,177 @@ casper-client put-deploy -n http://<NODE IP>:<PORT> \
+// This is the number of additional CEP-18 tokens to add to the total supply.
+--session-arg "amount:U256='10'" \
+--chain-name <CHAIN NAME> \
+--payment-amount 1000000000
+```
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+```bash
+casper-client put-deploy -n http://<NODE IP>:<PORT> \
+--secret-key ~/casper/demo/user_a/secret_key.pem \
+--session-package-name "cep18_contract_package_CEP18" \
+--session-entry-point "mint" \
+--session-arg "owner:key='account-hash-683f53f56926f54ef9584b07585b025c68415dc05f7b2e56749153574b83d5cd'" \
+--session-arg "amount:U256='10'" \
+--chain-name <CHAIN NAME> \
+--payment-amount 1000000000
+```
+
+</details>
+
+#### Burning Tokens
+
+If the contract allows for burning, the following command will burn a number of CEP-18 tokens directly from the provided account. This decreases the total supply of the token in question.
+
+```bash
+casper-client put-deploy -n http://<NODE IP>:<PORT> \
+--secret-key ~/casper/demo/user_a/secret_key.pem \
+--session-package-name "cep18_contract_package_CEP18" \
+--session-entry-point "burn" \
+// This is the account that the tokens will be burned from.
+--session-arg "owner:key='account-hash-683f53f56926f54ef9584b07585b025c68415dc05f7b2e56749153574b83d5cd'" \
+// This is the number of CEP-18 tokens to remove from the total supply.
+--session-arg "amount:U256='10'" \
+--chain-name <CHAIN NAME> \
+--payment-amount 1000000000
+```
+
+<details>
+<summary><b>Casper client command without comments</b></summary>
+
+```bash
+casper-client put-deploy -n http://<NODE IP>:<PORT> \
+--secret-key ~/casper/demo/user_a/secret_key.pem \
+--session-package-name "cep18_contract_package_CEP18" \
+--session-entry-point "burn" \
+// This is the account that the tokens will be burned from.
+--session-arg "owner:key='account-hash-683f53f56926f54ef9584b07585b025c68415dc05f7b2e56749153574b83d5cd'" \
+// This is the number of CEP-18 tokens to remove from the total supply.
+--session-arg "amount:U256='10'" \
+--chain-name <CHAIN NAME> \
+--payment-amount 1000000000
+```
+
+</details>
+
+### Changing Account Security Permissions
+
+The `change_security` entrypoint can be used by an account with `admin` access to alter the security level of other accounts.
+
+There are five security levels, with the strongest level taking precedence over any other assigned levels. In order of highest strength to lowest:
+
+1. `None` - `None` overrides other security levels and removes all admin, minting and burning access of an account.
+
+2. `Admin` - Allows the account full access and control over the CEP-18 contract.
+
+3. `MintAndBurn` - Allows the account to mint new tokens and burn existing tokens.
+
+4. `Burner` - The account can burn tokens.
+
+5. `Minter` - The account can mint new tokens.
+
+**Be aware that removing all admin accounts will lock out all admin functionality.**
+
+```bash
+casper-client put-deploy -n http://<NODE IP>:<PORT> \
```

Comment:
We need some help with this command, @deuszex. I initially got this error: 

`invalid CLValue error: unknown variant vec<key>, expected one of bool, i32, i64, u8, u32, u64, u128, u256, u512, unit, string, key, account_hash, uref, public_key, byte_list, byte_array_<NUM>, opt_bool, opt_i32, opt_i64, opt_u8, opt_u32, opt_u64, opt_u128, opt_u256, opt_u512, opt_unit, opt_string, opt_key, opt_account_hash, opt_uref, opt_public_key, opt_byte_list, opt_byte_array_<NUM>`.

So instead of a `Vec<Key>`, I guessed a string for account hashes. For example, the following command worked:

```
casper-client put-deploy \
--node-address https://rpc.testnet.casperlabs.io/  \
--chain-name casper-test \
--secret-key ~/KEYS/secret_key.pem \
--session-package-name "cep18_contract_package_Test Token" \
--session-entry-point "change_security" \
--session-arg "mint_and_burn_list:string='account-hash-1ed5a1c39bea93c105f2d22c965a84b205b36734a377d05dbb103b6bfaa595a7,account-hash-0ea7998b2822afe5b62b08a21d54c941ad791279b089f3f7ede0d72b477eca34'" \
--payment-amount "2000000000"
```

https://testnet.cspr.live/deploy/ca3c2af2197cb863004f8d18d5c38c59a25a76dd683e651b263c0c87545cbca4

Reply:
I did some further testing today. I gave account-hash-e70dbca48c2d31bc2d754e51860ceaa8a1a49dc627b20320b0ecee1b6d9ce655 permissions to mint and burn and I was able to mint successfully with it.

```
casper-client put-deploy \
--node-address https://rpc.testnet.casperlabs.io/  \
--chain-name casper-test \
--secret-key ~/KEYS/multi-sig/keys/user_2_secret_key.pem \
--session-package-name "cep18_contract_package_Test Token" \
--session-entry-point "change_security" \
--session-arg "mint_and_burn_list:string='account-hash-1ed5a1c39bea93c105f2d22c965a84b205b36734a377d05dbb103b6bfaa595a7,account-hash-0ea7998b2822afe5b62b08a21d54c941ad791279b089f3f7ede0d72b477eca34,account-hash-e70dbca48c2d31bc2d754e51860ceaa8a1a49dc627b20320b0ecee1b6d9ce655'" \
--payment-amount "2000000000"
```

https://testnet.cspr.live/deploy/3f1bb666961aa8437b58424d58e09be2834843125eec0a3a590f347404a2de7e

```
casper-client put-deploy \
--node-address https://rpc.testnet.casperlabs.io/  \
--chain-name casper-test \
--secret-key ~/KEYS/multi-sig/keys/user_2_secret_key.pem \
--session-package-name "cep18_contract_package_Test Token" \
--session-entry-point "mint" \
--session-arg "owner:key='account-hash-e70dbca48c2d31bc2d754e51860ceaa8a1a49dc627b20320b0ecee1b6d9ce655'" \
--session-arg "amount:u256='70'" \
--payment-amount "2000000000"
```

https://testnet.cspr.live/deploy/0ea9bd1a7936f26e9b7b523e4ff982f51a3c64e8a3f2fe27ff116cec6dbbefc2

Reply:
I'll play around a little later today and see if I can get a list of hashes in.

Reply:
Nevermind, I see that your first message had multiples! Awesome.

Thanks, @ipopescu!

Reply:
Updated to include this information.